Competition2017/src/competition/subsystems/drive/commands/RotateToHeadingCommand.java

```diff
@@ -9,10 +9,12 @@
 import xbot.common.math.PIDManager;
 import xbot.common.properties.XPropertyManager;
 import competition.subsystems.drive.DriveSubsystem;
+import competition.subsystems.pose.PoseSubsystem;
 
 public class RotateToHeadingCommand extends BaseCommand{
 
     final DriveSubsystem drive;
```

Comment:
These should be `private` or `protected`; we should avoid the default accessibility.